app/templates/components/summary-row.hbs

```diff
@@ -0,0 +1,18 @@
+{{#if payment.isDeleted}}
+{{else}}
```

Comment:
For showing summary for payments where isDeleted is false you can use `{{#unless}}`